manuscript.qmd

```diff
@@ -723,7 +723,7 @@ provenance of the integrated data object.
+\captionof{figure}{\textbf{Multi-modal data integration worklow.} Following data generation and preprocessing (top two panels), the integrative framework covers five steps of microbiome data integration and analysis: 1) Data is wrapped into a TreSE data object (see Figure~\ref{fig-datacontainers}), containing information on the samples (tube icons) and their features (e.g. taxonomic, functional, or resistome profiles; colored circles). TreeSE objects can accommodate multiple assay tables as well as additional hierarchical structures. 2) The TreeSE data object can then be processed to get cleaner data. This can include filtering out low-quality samples and features, transforming assay data for example by normalizing it to enhance the comparability of sample-specific abundances, or agglomerating the feature data to specific taxonomic levels or functional categories. 3) The cleaned data can then be analysed using statistical models to uncover relations among samples (e.g., with PCoA), analyse drivers of variation in community diversity and composition or test for differential abundance (e.g. between sampling time or location). 4) The framework supports the integration of various types of omics modalities through MultiAssayExperiment and the specialized data structures for each omics provided by Bioconductor. Here, for example, the taxonomic and metabolomic profiles of samples can be summarized and compared with joint analysis among features (e.g. taxa or metabolites). 5) Finally, Quarto notebooks can aid reproducible analysis and reporting.}
```

Comment:
> Here, for example, the taxonomic and metabolomic profiles of samples can be summarized and compared with joint analysis among features (e.g. taxa or metabolites).

Does this make sense? I would remove "(e.g. taxa or metabolites)" as it feels confusing